### DISCUSSION TITLE
clr-datagrid filter

### DISCUSSION THREAD

**Discussion by rdlepine** [vmware-clarity/core#315] [Q&A] [answered]
When using filters I can only enter 1 character.  I have to click on the filter again to enter subsequent characters. Am I missing some attribute? Seems to work that way on the Clarity web page code samples as well.
Thanks,
Ron

**Comment by rlepine** [comment]
Works with clr version 16.2. Using angular 17 and clr 17.2 it's not.

**Reply by rdlepine** [reply, marked as answer]
Just updated to 17.2.1 (just released). Working now.
